docs/Inflight Adjustments.md

```diff
@@ -12,16 +12,16 @@ Changing settings during flight can make your aircraft unstable and crash if you
 * Make small adjustments and fly carefully to test your adjustment.
 * Give yourself enough flying space and time to adjust to how your changes affect the behaviour of the aircraft.
 * Remember to set adjustment channel switches/pots to the center position before powering on your TX and your aircraft.
-* If possible configure switch warnings on your transitter for dedicated adjustment switches. 
+* If possible configure switch warnings on your transmitter for dedicated adjustment switches. 
 * A momentary 3 position switch is the best choice of switch for this - i.e. one that re-centers itself when you let go of it.
 
 ## Overview
 
 There are two modes of operation. The first supports adjusting settings by incrementing/decrementing them through use of an aux channel, typically a three position switch, where the middle position makes no change, and the other positions either increment or decrement the selected setting.
 
-The other mode is Absolute mode where a pot (knob/slider) may be directly mapped and changes the selected setting to a value based on a center value, coresponding to mid-postion on the pot and a range of ± adjustment at the min/max positions of the pot.
+The other mode is Absolute mode where a pot (knob/slider) may be directly mapped and changes the selected setting to a value based on a center value, corresponding to mid-postion on the pot and a range of ± adjustment at the min/max positions of the pot.
 
-Both modes utilise two channels to make an adjustment.
+Both modes utilize two channels to make an adjustment.
```

Comment:
"Utilise" is also correct. In Betaflight we often use the Queen's english

Reply:
Disagree again ;-)

Reply:
Both spellings are valid and that's a fact lol. 😄

Reply:
True, welcome to Babylonia :roll_eyes:

Reply:
Again, I prefer to revert this one, no change is the best change. Oh, and God Save the Queen!